### PR TITLE
Make a few minor contributions

### LINKED ISSUE #20508: content error in shell example for GPG signing articles
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key 

### What part(s) of the article would you like to see updated?

Slight inconsistency in examples section of following article (Image attached)
https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-gpg-key
in (3) and (4) point of Mac OS version
Screen Shot 2022-09-09 at 4 52 58 PM

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by applecuckoo**
Just correcting a small typo and changing key. Fixes https://github.com/github/docs/issues/20508

### Why:

Fixing a small typo in the docs and changing subkey in [paste-gpg-key-id.md](docs/data/reusables/gpg/paste-gpg-key-id.md) to match [copy-gpg-key-id.md](docs/data/reusables/gpg/copy-gpg-key-id.md) :memo:

### What's being changed (if available, include any code snippets, screenshots, or gifs):

```diff
- Once your PR is merged, your contributions will be publicly visible on the [GitHubs docs](https://docs.github.com/en). 
+ Once your PR is merged, your contributions will be publicly visible on the [GitHub docs](https://docs.github.com/en). 
```
```diff
- Alternatively, when setting a subkey include the `!` suffix. In this example, the GPG subkey ID is `4BB6D45482678BE3`:
+ Alternatively, when setting a subkey include the `!` suffix. In this example, the GPG subkey ID is `42B317FD4BA89E7A`:
```shell
- $ git config --global user.signingkey <em>4BB6D45482678BE3</em>!
+ $ git config --global user.signingkey <em>42B317FD4BA89E7A</em>!
```

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by applecuckoo** [comment]
Hey there folks, not too sure what the 'changes in staging' checkbox means so I've leaved that unticked.

**Comment by cmwilson21** [comment]
@applecuckoo Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: data/reusables/gpg/paste-gpg-key-id.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
    $ git config --global user.signingkey <em>3AA5C34371567BD2</em>
    ```
    
-   Alternatively, when setting a subkey include the `!` suffix. In this example, the GPG subkey ID is `4BB6D45482678BE3`:
+   Alternatively, when setting a subkey include the `!` suffix. In this example, the GPG subkey ID is `42B317FD4BA89E7A`:
```
Comment (jhosman) [inline data/reusables/gpg/paste-gpg-key-id.md:6]:
```suggestion
   Alternatively, when setting a subkey include the `!` suffix. In this example, the GPG subkey ID is `4BB6D45482678BE3`:
```

File: data/reusables/gpg/paste-gpg-key-id.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
    ```shell
-   $ git config --global user.signingkey <em>4BB6D45482678BE3</em>!
+   $ git config --global user.signingkey <em>42B317FD4BA89E7A</em>!
```
Comment (jhosman) [inline data/reusables/gpg/paste-gpg-key-id.md:8]:
```suggestion
   $ git config --global user.signingkey <em>4BB6D45482678BE3</em>!
```

**Review by jhosman — APPROVED** [review]
@applecuckoo Thanks so much for opening a PR! https://github.com/github/docs/issues/20508 has already been fixed, so I deleted those changes. I'll get your typo fix merged right after I update the branch and tests are passing. Thanks again for your contribution!

**Review by Marion455 — COMMENTED** [review]
[]()

**Comment by Starr79** [comment]
That was along Journey 🌻✨️🍃

**Comment by 1054074339** [comment]
Duplicate of #

**Pull request merged**